client/packages/common/src/ui/layout/tables/material-react-table/useBaseMaterialTable.tsx

```diff
@@ -121,10 +120,12 @@ export const useBaseMaterialTable = <T extends MRT_RowData>({
     table.setDensity(density.initial);
   };
 
+  const hasColumnFilters = columns.some(col => col.enableColumnFilter);
+
   const displayOptions = useTableDisplayOptions({
     tableId,
     isGrouped,
-    enableColumnFilters,
+    hasColumnFilters,
```

Comment:
@CarlosNZ - per other PR, use this instead of explicit enable/disable of the toolbar filter button